.travis.yml

```diff
@@ -48,18 +48,38 @@ env:
     # - AWS_SECRET_ACCESS_KEY
     - secure: "F9oY34UDI8aXuZ9nb6z3/ltlqrNs4RUQHDZHa2YSFoq0r2Rn272Yg7oHfPh3deYNKxtUOBwwcpiC/7G9MqhkuLQEYhqY/KowidG6gOxCZdTDEVS10C4GDEMGj64URlUCS722tQYMIpniiB6LW2kEOOcEuzC6JsVxS7BowUEVk7Ly8BTzrgNuGFN2bSphRyrJyrS3Reo5a1YI3joUMzI29t7fegefTIHHONKhZpIE86wFRk11nwvJdt54Eu9dVFwiEiGW9L9SnG2ph4UiqN6z3iQFCQ8HNvnf+BvNasdMBR3f85DPnG/syqOcQrrE1e5T88UXRxyUwR2ty1ad+ZvQPggwsoFs1FmdJH04pGn0Tgpo/AHNe5c67tvDbw2W4jlFJgVI/keoFvQigSM2DUuL1i0v48TkZw8lgcfasRjtJi7iDTC4qqjyMLkY6y2TRTVDG/86qxK3No7vvJZajzntU3c6MUttiSDubaHL5RuklQ+7MjN5Dp1cD09L3Qqnm9KcjWzbNoyqriBO84VU4aJFF72jJ18qADX/UUj7cxi0ShY19BHX2Vrp3pPSNiePfO7okUrg5Wc4eD7u4eC+aUVHboEX35NhBnfj7/ASrsltIANYFmNarv3tvd67JIsm/Wz3pge6YOSV1pBr2IsGk5UVIXdJ/mm6GVWFlJ9PiELDBGc="
 
+git:
+  depth: false
+
 install:
   - ./releng/travis-install.sh
   # Adjust the environment per travis-install.sh
   - PATH=~/bin:$PATH
   - source ~/.gimme/envs/latest.env
   - export DEV_KUBECONFIG=~/.kube/$(cat ~/kubernaut-claim.txt).yaml
+  # Setup for docs
+  - node --version
+  - nvm use 11
+  - node --version
+  - source ./.ci/travis-fix-env
+  - ./.ci/install-blc
 
 script:
   #- docker pull ${DEV_REGISTRY}/builder:latest
   - ./releng/travis-script.sh
+  - ./.ci/pr-build-website-preview master
+  - ./.ci/publish-website-preview master
+  - ./.ci/blc-website-preview master
 
 after_script:
   # release the kubernaut claim
   - ./releng/travis-cleanup.sh
   # - docker tag builder:latest ${DEV_REGISTRY}/builder:latest && docker push ${DEV_REGISTRY}/builder:latest
+
+# TODO: enable this post netlify testing
```

Comment:
Commented this right now, because this part publishes to getambassador.io.git which updates the website. Once we know that blc and web preview are working fine, will enable this.